examples/floyd-warshall/floyd_warshall.cc

```diff
@@ -18,14 +18,14 @@
 #include "FW-APSP/FloydRecursiveSerialKernel.h"  // contains the recursive but serial kernels
 // #include "FloydRecursiveParallelKernel.h" // contains the recursive and parallel kernels
 
-// needed for madness::hashT and xterm_debug
-#include <madness/world/world.h>
-
 #include "ttg.h"
 using namespace ttg;
 
+#include "ttg/serialization.h"
+#include "ttg/serialization/std/pair.h"
```

Comment:
Why not include `std/pair.h` et al. in `serialization.h? I need to remember to include it, otherwise compilation fails, right?

Reply:
include-what-you-use. although that train left the station eons ago (we use variadics and heavy metaprogramming everywhere in TTG) it seems to be unreasonable to include support for serialization of all `std` types